src/Tableau.Migration.App.Core/Tableau.Migration.App.Core.csproj

```diff
@@ -38,6 +38,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="System.Private.Uri" Version="4.3.2" />
```

Comment:
It doesn't look like you made any code changes to the app itself, is this required?